src/snowflake/snowpark/_internal/tcm/eval.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+# evaluate AST given as base64 encoded protobuf
+import argparse
+import base64
+import json
+import sys
+
+
+def process_ast(request_id: str, base64_encoded_ast: str) -> None:
```

Comment:
maybe `eval`? it seems like it should be the same as the name of the module, since both are trying to say the same thing.

Reply:
renamed to `eval_ast`. `eval` is a builtin global python function, I'd like to avoid a name clash.